src/dev/run_check_published_api_changes.ts

```diff
@@ -43,7 +43,18 @@ import getopts from 'getopts';
  */
 
 const getReportFileName = (folder: string) => {
-  return folder.indexOf('public') > -1 ? 'public' : 'server';
+  switch (true) {
+    case folder.includes('public'):
+      return 'public';
+    case folder.includes('server'):
+      return 'server';
+    case folder.includes('common'):
```

Comment:
`state_containers` are used on server also. so them are in common and we need API Extractor to support `common`